app/controllers/application_controller.rb

```diff
@@ -2,9 +2,9 @@ class ApplicationController < ActionController::Base
   include Pundit
   protect_from_forgery with: :exception
 
-  before_action :configure_permitted_parameters, if: :devise_controller?
+  # before_action :configure_permitted_parameters, if: :devise_controller?
   before_action :masquerade_user!
-  around_action :set_time_zone, if: :current_user
+  # around_action :set_time_zone, if: :current_user
```

Comment:
both these were breaking the log in. Had to comment out for now to be able to log in